gradle/built-uber-dists/NOTICE-BINARY-DIST

```diff
@@ -0,0 +1,446 @@
+
+Dremio
+Copyright 2015-2025 Dremio Corporation
+
+
+This product includes additional software licensed under the terms
+of the following licenses, see LICENSE (named LICENSE-BINARY-DIST in
+source distributions) file:
+* Apache Software License, Version 2.0
+* Creative Commons 1.0 Universal
+* Eclipse Distribution License, Version 1.0
+* Eclipse Public License, Version 1.0
+* Eclipse Public License, Version 2.0
+
+Some binary executables and containers contain software that is only licensed
+under the terms of the "GNU General Public License, Version 2 with the GNU
+Classpath Exception":
+* nessie-gc-tool contains com.github.pjfanning:jersey-json, transitively
+  via org.apache.hadoop:hadoop-common
+* nessie-quarkus contains javax.annotation:javax.annotation-api, transitively
+  via com.google.cloud:google-cloud-secretmanager
+* nessie-server-admin-tool contains javax.annotation:javax.annotation-api,
+  transitively via com.google.cloud:google-cloud-secretmanager
+* nessie-cli contains javax.annotation:javax.annotation-api, transitively
+  via com.google.cloud:google-cloud-storage
+
+For the full and concrete list of dependencies and code contained in Nessie
+binary distributions is contained in LICENSE for Nessie binary distributions
+(named LICENSE-BINARY-DIST in source distributions).
+
+The Nessie web site provides aggregated license reports for recent Nessie
+binary distributions, available at https://projectnessie.org/docs/ and
+https://projectnessie.org/downloads/.
+
+
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+---
```

Comment:
Use the same separator and general text style as the root NOTICE?

Reply:
i'll leave something to follow-up :P